ui/lib/theme.ts

```diff
@@ -26,6 +26,7 @@ export const theme: DefaultTheme = {
     primaryLight: "#98E0F7",
     primary10: "#009CCC",
     primary20: "#006B8E",
+    primary05: "#E5F7FD",
```

Comment:
Maybe pop it up the top to keep the ordering

```suggestion
    primary05: "#E5F7FD",
    primary10: "#009CCC",
    primary20: "#006B8E",
```

Reply:
Nice, is the color straight from figma?